src/agent/src/rpc.rs

```diff
@@ -139,10 +147,14 @@ fn is_allowed(req: &impl MessageDyn) -> ttrpc::Result<()> {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Derivative)]
+#[derivative(Clone, Debug)]
 pub struct AgentService {
     sandbox: Arc<Mutex<Sandbox>>,
     init_mode: bool,
+    #[derivative(Debug = "ignore")]
+    #[cfg(feature = "sealed-secret")]
+    cdh_client: Option<SealedSecretServiceClient>,
```

Comment:
In future there might be more APIs to be called by the client. We might need to impl all the RPCs in one Client, s.t. `CdhClient` or something else

Reply:
wrap SealedSecretServiceClient in to CDHClient in cdh.rs, thanks.